app/controllers/stripe-setup/bank-details/post.controller.js

```diff
@@ -35,20 +37,34 @@ module.exports = async (req, res, next) => {
     return response(req, res, 'stripe-setup/bank-details/index', {
       accountNumber: rawAccountNumber,
       sortCode: rawSortCode,
+      switchingToCredentials,
       errors
     })
   }
 
   try {
-    const stripeAccount = await connector.getStripeAccount(req.account.gateway_account_id, req.correlationId)
+    const switchingCredential = getSwitchingCredentialIfExists(req.account)
```

Comment:
Moving this variable into the scope of the if block might make it clearer when it's relevant. 

Also should we use the variation of this method that throws an exception if the switching credential does not exist - `getSwitchingCredential`, as we would expect it to if on the switching page?

Reply:
Really good catch -- I hadn't picked up on that but using the non wrapped method will make any edge cases much clearer